workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -254,6 +450,10 @@ private JSONObject _search(String jql, String[] returnFields)
 						"jql", jql
 					).queryParam(
 						"fields", StringUtil.merge(returnFields)
+					).queryParam(
+						"maxResults", maxResults
```

Comment:
also, pass this in as an integer. If needed, do the string.valueOf() at this point.